assets/javascript/app.js

```diff
@@ -0,0 +1,175 @@
+// array to store trivia questions
+var questions = [
+  {
+    question: "What house at Hogwarts does Harry belong to?",
+    A: "Gryffindor",
+    B: "Ravenclaw",
+    C: "Slytherin",
+    D: "Hufflepuff",
+    correctAsw: "A"
+  },
+  {
+    question: "What position does Harry play on his Quidditch team?",
+    A: "Bludger",
+    B: "Seeker",
+    C: "Chaser",
+    D: "Keeper",
+    correctAsw: "B"
+  },
+  {
+    question: "Who is Fluffy?",
+    A: "Hagrid's dragon",
+    B: "A three-headed dog",
+    C: "Hermione's cat",
+    D: "Harry's owl",
+    correctAsw: "B"
+  }
+];
+// array store images related to correct answers
+var imgUrl = [
+  "assets/images/hp_gryffindor.jpg",
+  "assets/images/hp_quidditch.png",
+  "assets/images/hp_fluffy.png"
+];
+// set global variables
+var maxTime = 20;
+var time = maxTime;
+var intervalId;
+var correctNum = 0;
+var wrongNum = 0;
+var noNum = 0;
+var i = 0; //get first question
+// when start button is clicked, clear contents of the page and then render the first question
+$("button.start").on("click", function() {
+  clear();
+  render();
+});
+// function to clear contents of the page
+function clear() {
+  $("div").html("");
+}
+// function to update timer text
+function updateTimerText() {
+  $(".timer").html(`<h3>Time Remaining ${time} Seconds</h3>`);
+}
+// function to start countdown
+function timer() {
+  clearInterval(intervalId);
+  updateTimerText();
+  intervalId = setInterval(decrement, 1000);
+}
+// function to decrease time by 1 until time is equals to 0
+function decrement() {
+  time--;
+  updateTimerText();
+  // when time equals to 0, clear contents of the page, add text and image related to correct answer
+  if (time === 0) {
+    clearInterval(intervalId);
+    clear();
+    var alert = $("<h3>").text("TIME IS UP!");
+    $(".reaction").append(alert);
+    var idx = questions[i].correctAsw;
+    var words = $("<h4>").text(`The Correct Answer was: ${questions[i][idx]}.`);
+    $(".reaction").append(words);
+    var img = $("<img>").attr("src", imgUrl[i]);
+    $(".reaction").append(img);
+    // update variable to record number of questions that were not answered
+    noNum++;
+    // reset timer
+    time = maxTime;
+    // move onto next question
+    i++;
+    // wait 5 seconds to render next question
+    setTimeout(clear, 5000);
+    setTimeout(render, 5000);
```

Comment:
Five seconds is a fairly long time to reset. It might be worth having a countdown displayed to the player so that they know they're waiting for the rest of the game!